util.c

```diff
@@ -485,7 +485,7 @@ bool should_bypass_proxy(const char *url, const char *bypass_list) {
         return true;
 
     // Strip and parse port from host
-    int32_t host_port = strip_host_port(host, strlen(host), 0);
+    uint16_t host_port = strip_host_port(host, strlen(host), 0);
```

Comment:
Can you move it closer to the usage?

Reply:
I prefer to keep it with the other host parsing code.